db/migrate/20180212230128_acts_as_taggable_on_migration.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,32 @@
+# This migration comes from acts_as_taggable_on_engine (originally 1)
+# rubocop:disable all
+class ActsAsTaggableOnMigration < ActiveRecord::Migration[4.2]
+  def self.up
+    create_table :tags do |t|
+      t.string :name
```

Comment:
Maybe we can add some not-null constraints to some of the columns in these tables?

Reply:
Question:  These were auto-generated during the installation of the acts-as-taggable-on gem... should we be modifying them to any real extent?

Reply:
That's good question, I think in situation like this it depends on what is the change you want to make to the tables.  If it's just adding not-null constraints, it's low risk that the gem will rely on the columns being able to be null, and the benefit of adding not-null constraints is huge because it will prevent any data corruption or whatnot.

Reply:
Ok, that makes sense.